content/docs/support-and-ops/ops-recipes/kyverno-stuck-upgrade-pending.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Kyverno Stuck In Pending Upgrade"
+owner:
+- https://github.com/orgs/giantswarm/teams/team-shield
+confidentiality: public
+---
+
+There have been cases where the cluster upgrades, for example from AWS v18 -> v19, provoked the security bundle and in particular Kyverno get stuck in Helm `pending-upgrade` till manual intervention.
```

Comment:
```suggestion
There have been cases where during cluster upgrades, for example from AWS v18 -> v19, the Kyverno migration logic takes longer than the default `app-operator` installation timeout. This can result in Kyverno getting stuck in Helm `pending-upgrade` and requiring manual intervention.
```